### PR TITLE
fix(PostInstall): update UI based on feedback

### PR DESCRIPTION
**Pull request by sidvishnoi**
## Context

Closes https://github.com/interledger/web-monetization-extension/issues/788

## Changes proposed in this pull request

- Turn the "Let's go" button to look more like step, but also make it look like button. Change copy to "Connect your wallet".
- Update welcome text copy.
- Add step number to steps.
- Ensure user can see there's more steps on screens with less height also (reduce image height).
- Update screenshots.
- Reduce layout shifts, improve responsive design.
- Embed connect wallet form (via popup as iframe) in Connect your wallet step.
   - Removed header from connect screen when used in iframe.
   - Follow up: https://github.com/interledger/web-monetization-extension/issues/793

### CODE REVIEW

File: src/assets/images/wallet-signup.png
Comment (sidvishnoi) [inline src/assets/images/wallet-signup.png:1]: IDK why this has a transparent outline 😆

Comment (sidvishnoi) [reply src/assets/images/wallet-signup.png:1]: But looks ok on screen.

**Review by sidvishnoi — COMMENTED** [review]
test-e2e

**Review by DarianM — APPROVED** [review]

**Review by raducristianpopa — APPROVED** [review]
🚀

**Pull request merged**